backend/src/attempts/attempts.controller.ts

```diff
@@ -29,8 +28,7 @@ export class AttemptsController {
   async userAttempt(
     @Req() request: Request,
     @Param('category_id') category_id: number,
-    @Param('word_id') word_id: number,
-    @Body('isDone') isDone: boolean
+    @Param('word_id') word_id: number
   ) {
     const cookie = request.cookies['jwt'];
     const { id: user_id } = await this.jwtService.verifyAsync(cookie);
```

Comment:
removed this.

Reply:
ah pls ignore. you can keep it.

but delete the following since we already have this in auth middleware.
![image](https://github.com/framgia/sph-els-mark/assets/73863685/070a1fff-e70e-4163-a555-8f71b9f44a5d)